package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@step-security/workflow-conclusion-action",
+  "version": "3.0.3",
+  "description": "GitHub action to get workflow conclusion.",
+  "keywords": [
+    "github",
+    "github actions"
+  ],
+  "homepage": "https://github.com/step-security/workflow-conclusion-action",
+  "bugs": {
+    "url": "https://github.com/step-security/workflow-conclusion-action/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/step-security/workflow-conclusion-action.git"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Technote",
+    "email": "technote.space@gmail.com",
+    "url": "https://technote.space"
+  },
+  "files": [
+    "dist",
+    "action.yml"
+  ],
+  "scripts": {
+    "build": "rm -rdf dist && rollup -c",
```

Comment:
do we need to add a script to generate dist?

Reply:
We can build through ncc as well, but due to pre-commits defined by husky It allows code only in camel-case format. Hence making it difficult for user to commit.